docs/content/concepts/metadata-tags/asset-metadata.mdx

```diff
@@ -22,6 +22,20 @@ How metadata is attached to an asset depends on the type of metadata being attac
 
 ---
 
+## Standard asset metadata entries
```

Comment:
My first thought when I looked at this was "I have no idea where I'm supposed to use these." After looking at some examples, I determined it was in the `metadata` parameter and is applicable to both definition and materialization metadata. I think adding some context here about that would be helpful.

Reply:
Definitely. Updated to add some intro sentences to this section.